proxy-auth.ks.in

```diff
@@ -14,7 +14,7 @@ timezone America/New_York
 rootpw qweqwe
 shutdown
 
-# Install @core, which will also pull in testpkg-http-core from the addon repo
+# Install @core, which will also pull in a-core from the addon repo
```

Comment:
I think "a" is a too short name to be meaningful when reading the code for first time, "testpkg" was IMO better. But I won't fight for this :-)